### PR TITLE
Add ability to blink LED

### PR DESCRIPTION
**Pull request by cpacey**
"Pins" class needs a new name, but that can be done separately.

### CODE REVIEW

File: src/LedImpl.cpp [hunk 1]
```diff
@@ -19,8 +19,40 @@ void LedImpl::setup() {
...
 
+#include <iostream>
+
+void LedImpl::loop() {
+  if( isBlinking() ) {
+    unsigned long timeSinceStart = pins->millis() - startMillis;
+    unsigned long period = delayOn + delayOff;
+    unsigned long pointInPeriod = timeSinceStart % period;
+
+    if ( pointInPeriod <= delayOn) {
```
Comment (jeffcharles) [inline src/LedImpl.cpp]: Maybe `delayOn` and `delayOff` could have clearer names. This doesn't read entirely clearly.

File: src/LedImpl.cpp [hunk 2]
```diff
@@ -19,8 +19,40 @@ void LedImpl::setup() {
...
 
+#include <iostream>
+
+void LedImpl::loop() {
+  if( isBlinking() ) {
+    unsigned long timeSinceStart = pins->millis() - startMillis;
+    unsigned long period = delayOn + delayOff;
+    unsigned long pointInPeriod = timeSinceStart % period;
+
+    if ( pointInPeriod <= delayOn ) {
```
Comment (jeffcharles) [inline src/LedImpl.cpp]: Should this be `<` instead? If `delayOn` were set to `0`, then pointInPeriod probably shouldn't trigger a blink. Granted, there's a check that `delayOn` isn't set to `0` around this.

Comment (cpacey) [inline src/LedImpl.cpp]: Better argument is on/off of 1/1, which will leave the Led always on :disappointed:.

**Comment by jeffcharles** [comment]
Looks good to me

**Pull request merged**